.github/workflows/nightly.yml

```diff
@@ -3,7 +3,8 @@ name: Nightly
 on:
   schedule:
     - cron: '0 23 * * *'
```

Comment:
we just figured out, that nigthly (cron) jobs won't be executed, since this is `adapters` branch, not our `main` branch - it's a GHA limitation. Please move this job into the cmake.yml file